kubernetes/apps/charts/sentry/values.yaml

```diff
@@ -0,0 +1,29 @@
+# Reference: https://raslasarslas.medium.com/how-to-deploy-sentry-on-a-kubernetes-cluster-using-helm-600db31d4486
+# Sentry chart: https://github.com/sentry-kubernetes/charts
+sentry:
+  user:
+    create: true
+    email: admin@sentry.local
+    existingSecret: sentry-secrets
+    existingSecretKey: admin-password
+  nginx:
+    enabled: false
+  ingress:
+    enabled: true
+    annotations:
+      kubernetes.io/ingress.class: nginx
+      kubernetes.io/ingress.allow-http: "false"
+      nginx.ingress.kubernetes.io/use-regex: "true"
+      cert-manager.io/cluster-issuer: letsencrypt-prod
+    hostname: sentry.calitp.org
+    tls:
+      - secretName: metabase-tls
+        hosts:
+          - sentry.calitp.org
+  mail:
+    backend: smtp
+    useTls: true
+    username: "apikey"
+    port: 587
+    host: "smtp.sendgrid.net"
+    from: "hunter@compiler.la"
```

Comment:
this email is probably not ok to use anymore

Reply:
I will ask the services team; I think they had said it was OK last time we asked about Airflow emails coming from this.